### ISSUE TITLE
Timeline is not painted over from the beginning to the timeline click place

### ISSUE THREAD

**Issue by ivansteff** [google/shaka-player#2424]
**Have you read the [FAQ](https://bit.ly/ShakaFAQ) and checked for duplicate open issues?**
Yes

**What version of Shaka Player are you using?**
2.5.9 compiled

**Can you reproduce the issue with our latest release version?**
Yes

**Can you reproduce the issue with the latest code from `master`?**
Yes

**Are you using the demo app or your own custom app?**
demo app

**What browser and OS are you using?**
Google chrome 80.0.3987.122

**What did you do?**

run a demo video on the [shaka Demo](https://shaka-player-demo.appspot.com/demo/#audiolang=ru-RU;textlang=ru-RU;uilang=ru-RU;asset=https://storage.googleapis.com/shaka-demo-assets/angel-one/dash.mpd;panel=HOME;build=compiled)
Rewind video by clicking on progress bar.

**What did you expect to happen?**
When rewinding, when you click on the timeline, I expected that the progress will be shaded from the beginning of the timeline to the place of the click.

**What actually happened?**

When rewinding, when you click on the timeline, the shaded part does not reach the start of the timeline. I attach the archive with the video.
[shaka bug.zip](https://github.com/google/shaka-player/files/4249089/shaka.bug.zip)

**Comment by joeyparrish** [comment]
That is working as we intended.  The filled area shows you only what's buffered.  When the area at the beginning of the timeline is not buffered, it will not be filled in.

Comparing this with the Chrome native controls on which ours are based, I see that those behave differently, and fill in the bar all the way to the beginning.

Is this something you'd like to be able to change in your application?  Are you using our UI in your app?

**Comment by ivansteff** [comment]
@joeyparrish Yes, I use your ui. I would like to make the timeline fill from the beginning to the place of the click =)

**Comment by joeyparrish** [comment]
I just discussed this with our product manager, and here's what we've decided to do:

1. Convert this to a feature request
2. Add a UI config field to choose between the current behavior in v2.5.9 and the behavior you requested (which matches Chrome native controls and YouTube UI)
3. Cherry-pick that change to v2.5.x with the default set to v2.5.9's behavior, for backward compatibility
4. Change the default for v2.6 to match Chrome's native controls

This new config would be available to you first in the master branch, then in our next bugfix release v2.5.10.  When v2.6.0 is complete, you will be able to use the default value instead of a custom one.

Does this help?
